src/olympia/abuse/templates/abuse/emails/CinderActionRejectVersion.txt

```diff
@@ -6,7 +6,7 @@ Our review found that your content violates the following Mozilla policy or poli
 
 Affected versions: {{ version_list }}
 
-Based on that finding, those versions of your {{ type }} have been disabled on {{ target_url }} and are no longer available for download from Mozilla Add-ons, anywhere in the world. Users who have previously installed those versions will be able to continue using them.
+Based on that finding, those versions of your {{ type }} have been disabled on {{ target_url }} and are no longer available for download from Mozilla Add-ons, anywhere in the world. {% if not is_addon_being_blocked and not target.block %}Users who have previously installed those versions will be able to continue using them.{% endif %}
 
-You may upload a new version which addresses the policy violation(s).
+{% if not is_addon_being_disabled and not target.is_disabled %}You may upload a new version which addresses the policy violation(s).{% endif %}
```

Comment:
imo it'd be better if we weren't accessing `.block` and `.is_disabled` directly in the template.  
e.g. `'is_addon_blocked': log_entry.details.get('is_addon_being_blocked') or target.block`

Reply:
But... we'd be always making that query, even though we only need it for the reject version template.

Reply:
I'll need to remove the `.block` condition because of that `else` clause anyway, so maybe I can do that just for the disabled part (and call the corresponding variable `is_addon_disabled`)